README.md

```diff
@@ -1,6 +1,130 @@
+
+ 
+### **Telemetry**
+* Flows.
+* DNS (pcaps).
+
+### **Parallel Ingest Framework**
+* Open source decoders.
+* Load data in Hadoop.
+* Data Transformation.
+
+
+### **Machine Learning**
+* Filter billion of events to a few thousands.
+* Unsupervised learning.
+
+### **Operational Analytics**
+* Visualization.
+* Attack heuristics.
+* Noise filter.
+
+
+## **Getting Started**
+
+ONI can be installed by following our installation manual. To get started, [check out the installation instructions in the documentation](https://github.com/Open-Network-Insight/open-network-insight/wiki).
+
+
+## If you want to get all the ONI code at one just clone it !
+        git clone --recursive https://github.com/Open-Network-Insight/open-network-insight.git
+
+## **Roadmap**
+
+TBD by RITU
+
+## **Documentation (Developer Guide)**
+
+ONI functionality is divided into different repositories, go to each repository for developer documentation:
+
+* [oni-ingest](https://github.com/Open-Network-Insight/oni-ingest)
+* [oni-ml](https://github.com/Open-Network-Insight/oni-ml)
+* [oni-oa](https://github.com/Open-Network-Insight/oni-oa)
+* [oni-setup](https://github.com/Open-Network-Insight/oni-setup)
+* [oni-nfdump](https://github.com/Open-Network-Insight/oni-nfdump)
+* [oni-lda-c](https://github.com/Open-Network-Insight/oni-lda-c)
+
+## **Community Support**
+
+Our Central repository for our Open Network Insight solution is found here. If you find a bug, have question or something to discuss please contact us:
+
+* [Create an Issue](https://github.com/Open-Network-Insight/open-network-insight/issues). if you need more details about how to create an issue in ONI please go to: [How to create and Issue](ISSUES.md)
+* [Go to our Slack channel](https://opennetworkinsights.slack.com/messages/general/). 
+
+## **Contributing to ONI**
+
+Help us to improve ONI!
+
+ONI is Apache 2.0 licensed and accepts contribuitions via GitHub pull requests. Please follow the next steps
+and join our comunity.
+
+### **Contribuiting to ONI code**
+
+* Fork it, Indentify the module that you want to contribuite and for the corresponding repo.
+* Create a Branch, we use [topic branches](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows#Topic-Branches) for our commits. 
+* Push your commit(s) to your repository.
+* Create a pull request to the original repo in ONI organization.
+
+### **Commit Guidelines**
+
+* If you are doing bug fixing, it should be a single commit.
+* Please be clear with the commit messages about what you are fixing or adding to the code base. If you code is addressing an open issue please add the reference to the issue in the comments with: Fix: Issue's URL. 
+
+
+### **Merge approval**
+
+ONI maintainers use LGTM (Looks Good to Me) in a comments on the code review to indicate acceptance, 
```

Comment:
if this is standard, I guess I can live with it

the more I read LGTM the more I get used to it